introduction.md

```diff
@@ -5,4 +5,5 @@ Ruby, JavaScript
 Skills:
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?